src/containers/Application.js

```diff
@@ -20,12 +20,14 @@ import { Spinner } from './Spinner';
 import AlertsComponent from '../alerts/AlertsComponent';
 import AlertsReducer from '../alerts/reducer';
 
+import helpButtonReducer from '../helpButton/reducer';
```

Comment:
Well, I think it should stay with lowercase because reducers are pure functions, not a component or class. But I can rename other reducers with lower camel case

Reply:
Hmms, yes, you are right